### ISSUE TITLE
KasmVNC Integration

### ISSUE THREAD

**Issue by ProfessorManhattan** [megabyte-labs/install.doctor#56]
### 💡 Feature/Idea

KasmVNC provides enhanced VNC features when coupled alongside Kasm Workspaces which is what the project is currently evolving towards. See: https://kasmweb.com/

We are currently installing TigerVNC which is configured in the `/workspaces/install.doctor/home/.chezmoiscripts/universal/run_onchange_after_16-vnc.sh.tmpl` file.

KasmVNC has no easy way of getting installed I believe. We could hardcode each of the `.deb` and `.rpm` files and add them to `software.yml`. So that's the first thing that needs to be done.

Next, when both TigerVNC and KasmVNC are installed, we need to ensure that the `vncserver` and all the other related programs like `vncconfig` point to KasmVNC. I started adding logic to the `run_onchange_` file for KasmVNC which should give you a better idea on how to detect when they are installed.

**Testing:** Next, we need to ensure that we can connect to KasmVNC with a regular VNC client. It should work just like TigerVNC and then upgrade to the extra Kasm-related features if available (which should happen automatically).

Finally, we need to ensure the `systemd` files in the `dot_config/vnc/etc` folder is compatible for both KasmVNC and TigerVNC.

Basically, it would be awesome if we could replace TigerVNC with KasmVNC but that's not possible yet because KasmVNC only supports a limited number of Linux machines.

**Comment by enggnr** [comment]
Native VNC support is not available. It may work for now but it will break in the future. Please see [this](https://github.com/kasmtech/KasmVNC/issues/152), [this](https://github.com/kasmtech/KasmVNC/issues/132), [this](https://github.com/kasmtech/KasmVNC/issues/59) and [this](https://github.com/kasmtech/KasmVNC/issues/119).
